async_upnp_client/cli.py

```diff
@@ -114,11 +115,14 @@ def on_event(service: UpnpService, service_variables: Sequence[UpnpStateVariable
         'service_type': service.service_type,
         'state_variables': {sv.name: sv.value for sv in service_variables},
     }
-    print(json.dumps(obj, indent=pprint_indent))
+
+    is_last_change = len(service_variables) == 1 and service_variables[0].name == 'LastChange'
+
+    if not args.nolastchange or not is_last_change:
```

Comment:
Can you rewrite this? This seems to be more complex than needed and isn't easily readable, in my opinion.

Reply:
Hopefully the new version is easier to read, though I'm not sure it's any better logically.